src/merge-analysis/into-classes.js

```diff
@@ -91,8 +92,15 @@ function intoClasses(classes, members) {
 
   // Generate the class code
   for (const pair of values(pairs)) {
+    // This can happen when your documented members are all marked as private OR something is not documented.
     if (pair.members.length === 0) {
-      info(pair.class, 'Class', `path ${pair.path} has no defined members`);
+      info(pair.class, 'Class', `path ${pair.path} has no defined members, nothing will be documented.`);
+    }
+
+    // Do not add private classes
+    if (pair.class.comment.includes("@private")) {
+      error(pair.class, pair.class.TSDef, `${pair.path} contains a @private tag`)
```

Comment:
What is the remedy should we get such an error (I'm getting 4 of these)?

Reply:
My thought was that we should not be documenting these classes if we've got them tagged as private. So I think its either:
 1) remove the tag
 2) alter the generator code that we don't push the classes in
I'm still pushing them in b/c I think we're using them as interfaces and they are necessary

Reply:
Since it's coming up in 4 cases, what do you propose we do in these 4 specific cases?  If the answer is nothing, then we need a way to suppress these warnings for these 4 cases.

Reply:
For these specific cases I propose we remove the `@private` tag since we've already been generating them this way.

Reply:
I see.  So this line is catching any class with a `@private` tag.